.devcontainer/install-dependencies.sh

```diff
@@ -99,6 +111,7 @@ fi
 
 # Ensure we have AZ
 
+## | AZ | latest | https://docs.microsoft.com/en-us/cli/azure/install-azure-cli |
```

Comment:
Missing as well (kudos to @super-harsh for actually noticing this)

Reply:
Fixed.